api/api.go

```diff
@@ -24,6 +24,12 @@ type SplitioAPIBinding struct {
 	splitioAPIUri string
 }
 
+// Segment contains segment name and a slice of added values
+type Segment struct {
```

Comment:
The reason why I still create our own `Segment` is that we eventually only want (`Name`, `Added`) info for a segment. (want to do something similar to node version: https://github.com/godaddy/split-node-serializer/blob/master/lib/data-serializer.js#L17)

splitio sdk struct contains more variables( link: https://github.com/splitio/go-client/blob/master/splitio/service/dtos/segment.go), I also use that but want another type here as a returned type

Reply:
That's ok -- i think we should still use the split sdk version.

Reply:
Do you mean we use split sdk version as the returned type? by this way Go version will return more info than node.
and if yes, for extra fields I will set it as below:
```
Removed: [] always be empty
Since: the last since we got from this segment
Till: the last till we got from this segment
```
(in this case Since will equal to Till)

Reply:
Yes, use the split io SDK data types.